OpenRA.Mods.Common/Traits/Power/Player/PowerManager.cs

```diff
@@ -127,8 +128,9 @@ void UpdatePowerState()
 			if (isLowPower != wasLowPower)
 				UpdatePowerRequiringActors();
 
+			// Force the notification to play immediately
 			if (isLowPower && !wasLowPower)
-				nextPowerAdviceTime = 0;
+				lastPowerAdviceTime -= info.AdviceInterval;
```

Comment:
Was `lastPowerAdviceTime -= info.AdviceInterval;` versus `lastPowerAdviceTime = -info.AdviceInterval;` here intentional?

Reply:
Yes, though I realize now that your proposal essentially would have the same effect without a subtract operation.